chemprop/v2/cli/convert.py

```diff
@@ -0,0 +1,55 @@
+import sys
+import torch
+import logging
+from pathlib import Path
+from argparse import ArgumentParser, Namespace
+
+from chemprop.v2.cli.utils import Subcommand
+from chemprop.v2.utils.v1_to_v2 import convert_model_v1_to_v2_file
+
+class ConvertSubcommand(Subcommand):
+    COMMAND = "convert"
+    HELP = "convert a v1 model checkpoint (.pt) to a v2 model checkpoint (.ckpt)"
+
+    @classmethod
+    def add_args(cls, parser: ArgumentParser) -> ArgumentParser:
+        return add_convert_args(parser)
+
+    @classmethod
+    def func(cls, args: Namespace):
+        args = process_args(args)
+        main(args)
+
+def add_convert_args(parser: ArgumentParser) -> ArgumentParser:
+    parser.add_argument(
+        "-i",
+        "--input_path",
```

Comment:
It could be good to keep to our v2 convention to use dashes for long args `--input-path`.